Directory.Packages.props

```diff
@@ -0,0 +1,46 @@
+<Project>
+  <ItemGroup>
+    <PackageVersion Include="FSharp.Core" Version="5.0.2" />
+    <PackageVersion Include="Microsoft.CodeCoverage" Version="16.11" />
+    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="16.11" />
+    <PackageVersion Include="Microsoft.NETCore.Platforms" Version="5.0.2" />
+    <PackageVersion Include="Microsoft.NETCore.Targets" Version="5.0" />
+    <PackageVersion Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" />
+    <PackageVersion Include="Microsoft.TestPlatform.ObjectModel" Version="16.11" />
+    <PackageVersion Include="MSTest.TestAdapter" Version="2.2.7" />
+    <PackageVersion Include="MSTest.TestFramework" Version="2.2.7" />
+    <PackageVersion Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageVersion Include="NuGet.Frameworks" Version="5.11" />
+    <PackageVersion Include="NUnit" Version="[3.13.1,4.0.0)" />
```

Comment:
This is not a straight lift from paket.lock. It matches the current FsCheck.NUnit dependencies
<img width="791" alt="Screenshot 2024-02-24 at 18 04 04" src="https://github.com/fscheck/FsCheck/assets/3138005/f0eab0e2-669d-4ec9-9774-007adbf97167">